generated/nifgen/nifgen/session.py

```diff
@@ -1629,30 +1629,13 @@ def _create_waveform_f64_numpy(self, waveform_data_array):
 
         Example: :py:meth:`my_session._create_waveform_f64`
 
-        Args:
-            waveform_data_array (numpy.array(dtype=numpy.float64)): Specifies the array of data you want to use for the new arbitrary
-                waveform. The array must have at least as many elements as the value
-                that you specify in **waveformSize**.
-
-                You must normalize the data points in the array to be between –1.00 and
-                +1.00.
-
-                **Default Value**: None
-
-
         Returns:
             waveform_handle (int): The handle that identifies the new waveform. This handle is used later
                 when referring to this waveform.
 
         '''
         import numpy
 
-        if type(waveform_data_array) is not numpy.ndarray:
```

Comment:
This should have been caught by system tests. Are we missing coverage for these cases where the type is incorrect?

Reply:
Yes, I tried commenting out this block and all of the tests still pass so we are missing coverage. I created #1847 

Marking this thread resolved, since this only affects local tests (outside the scope of the current PR) and a separate issue has been created.